spec/workers/requeue_content_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe RequeueContent do
+  let(:edition) { FactoryGirl.create(:live_edition, base_path: '/ci1') }
+
+  describe "#call" do
```

Comment:
Might be good to change this from `#call`

Reply:
good call..